### PR TITLE
List metrics forwarded to the sumologic

### PR DESCRIPTION
**Pull request by sumo-drosiek**
###### Description

Add list of metrics which we forward to the sumologic

###### Testing performed

- [ ] ci/build.sh
- [ ] Redeploy fluentd and fluentd-events pods
- [ ] Confirm events, logs, and metrics are coming in

### CODE REVIEW

File: deploy/helm/sumologic/values.yaml [hunk 1]
```diff
@@ -729,24 +770,83 @@ prometheus-operator:
...
+        # prometheus_remote_storage_shards
+        # prometheus_remote_storage_shards_desired
+        # prometheus_remote_storage_shards_max
+        # prometheus_remote_storage_shards_min
+        # prometheus_remote_storage_string_interner_zero_reference_releases_total
+        # prometheus_remote_storage_succeeded_samples_total
+        # up
         - url: http://$(CHART).$(NAMESPACE).svc.cluster.local:9888/prometheus.metrics
           writeRelabelConfigs:
             - action: keep
```
Comment (frankreno) [inline deploy/helm/sumologic/values.yaml:852]: missing fluentd/bit/otel?

**Comment by sumo-drosiek** [comment]
Solves #603

File: deploy/helm/sumologic/values.yaml (hunk 1 above)
Comment (vsinghal13) [reply deploy/helm/sumologic/values.yaml:852]: +1

Comment (sumo-drosiek) [reply deploy/helm/sumologic/values.yaml:852]: Yeah, I was basing on metrics test. Need to run setup with logs and tracing to do full completion

**Comment by sumo-drosiek** [comment]
I updated list of metrics with fluentd, fluent-bit and otelcol :)

**Pull request merged**